app/presenters/hyrax/iiif_manifest_presenter.rb

```diff
@@ -93,12 +93,8 @@ def manifest_url
     ##
     # @return [Array<#to_s>]
     def member_ids
-      case model
-      when Valkyrie::Resource
-        Array(model.try(:member_ids))
-      else
-        Hyrax::SolrDocument::OrderedMembers.decorate(model).ordered_member_ids
-      end
+      m = model.is_a?(::SolrDocument) ? model.hydra_model : model
+      m.class < Hyrax::Resource ? Array(model.member_ids) : Hyrax::SolrDocument::OrderedMembers.decorate(model).ordered_member_ids
```

Comment:
what is `m.class < Hyrax::Resource` doing?

Reply:
Checks if the model's class is a subclass of Hyrax::Resource. [documentation](https://devdocs.io/ruby~3.2/module#method-i-3C)